elyra/pipeline/airflow/provider_package_catalog_connector/airflow-provider-package-catalog.json

```
@@ -0,0 +1,74 @@
+{
+  "$schema": "https://raw.githubusercontent.com/elyra-ai/elyra/master/elyra/metadata/schemas/meta-schema.json",
+  "$id": "https://raw.githubusercontent.com/elyra-ai/elyra/master/catalog-connectors/airflow/airflow-provider-package-catalog-connector/airflow-provider-package-catalog.json",
```

Just noticed this same thing in #2437 

```suggestion
  "$id": "https://raw.githubusercontent.com/elyra-ai/elyra/master/elyra/pipeline/airflow/provider_package_catalog_connector/airflow-provider-package-catalog.json",
```